dist/js/material.js

```diff
@@ -91,8 +91,11 @@
       // Add fake-checkbox to material checkboxes
       var $input = $((selector) ? selector : this.options.togglebuttonElements)
         .filter(":notmdproc")
-        .data("mdproc", true)
-        .after("<span class='toggle'></span>");
+        .data("mdproc", true);
```

Comment:
The purpose of `.filter(":notmdproc")` is to applu the following lines only if they have not been already processed.

This PR just ignores the fact that `:notmdproc` seems to not work properly and adds an ugly fix...